tests/test_special.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+from mssqlutils import create_mssql_cli_client, shutdown
+from mssqlcli.packages.special.main import special_command, execute, NO_QUERY
+
+# All tests require a live connection to a SQL Server database
+class SpecialCommandsTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        try:
+            # create the database objects to test upon
+            client = create_mssql_cli_client()
+            list(client.execute_single_batch_query('CREATE DATABASE testdb1;'))
```

Comment:
>testdb1 [](start = 68, length = 7)

can this by dynamic, eg. mssql-cli-test-xxxx-xxxxx

This way, concurrently running tests can't step on each other.